### DISCUSSION TITLE
Show background and/or skin credits in main menu

### DISCUSSION THREAD

**Discussion by revelsix** [ppy/osu#28847] [Ideas]
I noticed the same issue as #18827, but rather than adjust the parallax to fit an artist's signature I propose having some small text on the screen with the ability to click on the artist's name and view their osu! profile if available. Here's a quick mockup of what I mean :) 
![osu credit](https://github.com/user-attachments/assets/79ff2dfc-e562-46b4-aa46-56f0c356ce87)

**Comment by myutin** [comment]
Would love to see something like this added!
